core/core.mk

```diff
@@ -22,6 +22,7 @@ aflags$(sm)	+= $(platform-aflags) $(core-platform-aflags)
 # Config flags from mk/config.mk
 cppflags$(sm) += -DCFG_TEE_FW_DEBUG=$(CFG_TEE_FW_DEBUG)
 cppflags$(sm) += -DCFG_TRACE_LEVEL=$(CFG_TEE_CORE_LOG_LEVEL)
+cppflags$(sm) += -DCFG_MEASURE_PERFORMANCE=$(CFG_TEE_MEASURE_PERFORMANCE)
```

Comment:
And remember: with all variables named CFG_<something>, setting cppflags is not needed. `conf.h` defines the value.